curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-data-from-post-requests.english.md

```diff
@@ -7,7 +7,13 @@ challengeType: 2
 ## Description
 <section id='description'>
 Mount a POST handler at the path <code>/name</code>. It’s the same path as before. We have prepared a form in the html frontpage. It will submit the same data of exercise 10 (Query string). If the body-parser is configured correctly, you should find the parameters in the object <code>req.body</code>. Have a look at the usual library example:
-<blockquote>route: POST '/library'<br>urlencoded_body: userId=546&bookId=6754 <br>req.body: {userId: '546', bookId: '6754'}</blockquote>
+
+```js
```

Comment:
I'm not sure about this one - it doesn't look like it's supposed to be code.  Well, the last line `req.body: {userId: '546', bookId: '6754'}` is, but not the rest.  Maybe leave it as a blockquote.

Reply:
I reverted this one back to blockquote just now.